doc/python/plotly-express.md

```diff
@@ -53,6 +53,8 @@ print(px.data.iris.__doc__)
 px.data.iris().head()
 ```
 
+To see the full list of built-in datasets, click [here](https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/plotly/data/__init__.py).
```

Comment:
The best thing would probably be to add an `__all__` to `data/__init__.py` and to `print(px.data.__all__)`.